modules/vba_documents/spec/jobs/upload_processor_spec.rb

```diff
@@ -451,7 +444,7 @@
     end
 
     context 'with invalid sizes' do
-      %w[21x21 18x22 22x18].each do |invalid_size|
```

Comment:
21" x 21" is not an invalid size according to our documentation. It is the upper limit, but an inclusive upper limit.